next.clayui.com/src/utils/getPackageConfig.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {readFileSync} = require('fs');
+const {join, resolve} = require('path');
+
+function getPackageConfig(name) {
+	const internalPackagePrefix = '@clayui/';
+	if (name && name.startsWith(internalPackagePrefix)) {
+		const suffix = name.slice(internalPackagePrefix.length);
+		return readFileSync(
```

Comment:
New line above this return. We need to make sure we have new lines between any non-sortable content.